docs/csharp/language-reference/operators/patterns.md

```diff
@@ -72,6 +76,40 @@ The discard pattern is useful in `switch` expressions. If none of a `switch` exp
 
 The discard pattern cannot be a pattern in an `is` expression or a `switch` statement.
 
+## Relational patterns
```

Comment:
In this section, is it worth highlighting that the right side of the expression *must* be a constant? That restriction is the reason relational patterns are limited to the builtin numeric types. (Specifically, user defined types that define relational operators can't be used in relational patterns).

Reply:
@BillWagner to confirm I understand you correctly: you suggest to explicitly mention that the right-hand side must be a constant? If yes, I agree and will update accordingly.

Reply:
@BillWagner I've updated the paragraph after the first example. Please check how it reads now.